### DISCUSSION TITLE
Version 8.2.0.23 Slow Download Speed & Unable to Download Multiple Files Simultaneously

### DISCUSSION THREAD

**Discussion by RighteousHundred** [aclap-dev/video-downloadhelper#125] [Help and Q&A]
### Your question

Two days ago, following the automatic update to version 8.2.0.23, the download speed has significantly reduced, and I am no longer able to download multiple files simultaneously. Also, the newly introduced audio-only download feature is taking a very long time to download. I really liked your previous version of Video DownloadHelper because it was fast, and I could download even 20 files at the same time. Please look into this issue. Thank you.

**Comment by jcfrog** [comment]
Hi,
Sorry 8.2.0.23 won't live a long time. An update is coming soon.
